go/types/encode_human_readable.go

```diff
@@ -195,6 +204,23 @@ func (w *hrsWriter) writeStruct(v Struct, printStructName bool) {
 	w.write("}")
 }
 
+var dateTimeType = MakeStructTypeFromFields("DateTime", FieldMap{
+	"secSinceEpoch": NumberType,
+})
+
+func (w *hrsWriter) FormatDateTime(v Value) string {
```

Comment:
I thought that there was a bug for this, and that we decided to do it with comments, like so:

```
struct DateTime {  // 2017-07-31T20:36:00
  msSinceEpoch: 1501558457708,
}
```

... it's not as beautiful as it could be, but it's nice because it's general (other things could have similar comments)